src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/VersioningTests.cs

```diff
@@ -1,22 +1,26 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
 using System.Net;
+using System.Net.Http;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Mvc.FunctionalTests
 {
-    public class VersioningTests : VersioningTestsBase<VersioningWebSite.StartupWith21Compat>
+    public class VersioningTests : IClassFixture<MvcTestFixture<VersioningWebSite.Startup>>
```

Comment:
What I did here is more like renaming `VersioningTestsBase` to `VersioningTests` and using the `VersioningEndpointRoutingTests` choice of `Startup` class. This one is slightly messy because `VersioningEndpointRoutingTests` contained a few `override`s.